bin/pgenv

```diff
@@ -1168,6 +1169,15 @@ case $1 in
         exit
         ;;
 
+    status)
+        # only proceed when an instance is loaded
+        pgenv_exit_if_no_postgresql_in_use
+        # print standard postgres status
+        $PG_CTL status -s -D "$PG_DATA" | sed -n '/^pg_ctl:\s/s/pg_ctl:\s\(.*\)/\1/p'
```

Comment:
Should `\s\s` be `\s+`?

Reply:
Actually it's not `\s\s`, sed will only search for lines matching the pattern we write like `sed '/pattern/command'`. So this is the pattern `^pg_ctl:\s` and the command `'s/match/replace/'`, combined as `sed '/pattern/s/match/replace/'`.
Although I'm pretty sure there is only one space in the pattern, using `^pg_ctl:\s+` don't hurt.
I don't have access to my laptop now, I'll push the changes soon.